apps/deno/tests/test18.ts

```diff
@@ -0,0 +1,27 @@
+import { Assets } from '../index.ts';
```

Comment:
Can we update this test to include all the field types? I'd like to be able to verify that the form flattening feature works for all of them.

Also, it would be nice to verify that once a form is flattened, it's filled "fields" (or, more precisely, their appearances) can be copied into other documents via normal page copying. So perhaps this test could load the documents used in tests 14-17, fill some fields in each, flatten them, and then copy a subset of the pages from each of them into a new document.